controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/RoutingPolicy.java

```diff
@@ -6,22 +6,21 @@
 import com.yahoo.config.provision.ClusterSpec;
 import com.yahoo.config.provision.HostName;
 import com.yahoo.config.provision.RotationName;
+import com.yahoo.config.provision.SystemName;
 import com.yahoo.config.provision.zone.ZoneId;
+import com.yahoo.vespa.hosted.controller.application.Endpoint.Port;
 
-import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.Set;
-import java.util.function.Predicate;
-import java.util.stream.Collectors;
 
 /**
  * Represents the DNS routing policy for a load balancer.
  *
  * @author mortent
  * @author mpolden
  */
-public class RoutingPolicy {
+public class  RoutingPolicy {
```

Comment:
typo I guess?

Reply:
Already fixed in 4fac9aefba8b590a5b190ae42cdd7ae9a3fc38e9.